code/client/src/pages/Show.jsx

```diff
@@ -131,11 +134,30 @@ const Show = () => {
     const index = ONEUtil.timeToIndex({ effectiveTime })
     const neighbors = ONE.selectMerkleNeighbors({ layers, index })
     const neighbor = neighbors[0]
+
+    // Check for one address
+    let verifiedTransferTo = transferTo
```

Comment:
Can be merged with above using utility function

Reply:
For sure, too much duplication. Will fix.